app/content/models/community_model.rb

```diff
@@ -1,4 +1,4 @@
 class CommunityModel < Sitepress::Model
   collection glob: "**/community/*.html*"
-  data :title
+  data :title, :breadcrumb
```

Comment:
What do we need the `:breadcrumb` here in all the Sitepress models for?

Reply:
correct, we didn't need that.